src/controllers/wallet/wallet.controller.js

```diff
@@ -763,6 +763,42 @@ async function stop(req, res) {
   res.send({ success: true });
 }
 
+/**
+ * Mark the inputs from the txHex as used.
+ *
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+async function markUtxosSelectedAsInput(req, res) {
+  const validationResult = parametersValidation(req);
+  if (!validationResult.success) {
+    res.status(400).json(validationResult);
+    return;
+  }
+
+  /** @type {{ wallet: import('@hathor/wallet-lib').HathorWallet }} */
+  const { wallet } = req;
+  const { txHex, ttl } = req.body;
+  const markAsUsed = req.body.mark_as_used ?? true;
+
+  try {
+    const tx = helpersUtils.createTxFromHex(txHex, wallet.getNetworkObject());
+    await Promise.all(
+      tx.inputs.map(
+        input => (wallet.storage.utxoSelectAsInput(
```

Comment:
Using storage methods directly seems not good to me, we have a method in the wallet class `markUtxoSelected` that we could use for that. The only downside is that we don't have the `ttl` parameter there.

What do you think would be better? 

1. Remove the `ttl` parameter here.
2. Add the `ttl` parameter in the lib.
3. Keep using the `storage` method directly?

Reply:
For me it's good